tests/e2e/tests/packages/webpack/test.ts

```diff
@@ -6,8 +6,6 @@ import {expectFileSizeToBeUnder} from '../../../utils/fs';
 
 
 export default function(argv: any, skipCleaning: () => void) {
-  const currentDir = process.cwd();
```

Comment:
This is needed, because the file uses this variable to switch directories after that (or maybe change https://github.com/angular/angular-cli/pull/3287/files#diff-826d2a48a6d98e1c11c9d7014d637925L27 to read from env variables).